tests/performance/tests/checkout-spike.js

```diff
@@ -0,0 +1,22 @@
+import { CheckoutFlow } from '../specs/shopper/checkout.js';
+
+export let options = {
+    scenarios: {
+      checkoutFlowSpike: {
+        executor: 'ramping-arrival-rate',
```

Comment:
May I suggest using `ramping-vus` executor instead. It's bit more easier to get started with, has less configurations and its also easier to make sense of results. For context, I am testing with this config:

```javascript

export let options = {
    scenarios: {
      checkoutFlowSpike: {
        executor: 'ramping-vus',
        startVUs: 1,
        stages: [
          { target: 1, duration: '120s' },
          { target: 10, duration: '120s' },
          { target: 25, duration: '120s' },
          { target: 75, duration: '120s' },
          { target: 100, duration: '120s' },
        ],
        gracefulRampDown: '30s',
        exec: 'checkout',
      },
    },
  };
```
Then when testing, it will allow people to add/remove stages till they hit a state where performance degrades severely. wdyt?

Reply:
Yes I think this needs more experimenting, `ramping-vus` does look good too.
For now I've added a additional test scenario file `simple-all-requests.js` just for the purpose of testing and debugging the requests.